### PR TITLE
Handle tagged update info on release branch

### PR DESCRIPTION
**Pull request by mdrohmann**
https://www.pivotaltracker.com/story/show/179257308

### CODE REVIEW

File: cmd/state/forward.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const LatestVersion = "latest"
 
 type forwardFunc func() error
 
-func forwardFn(bindir string, args []string, out output.Outputer, pj *project.Project) (forwardFunc, error) {
+func forwardFn(cfg updater.Configurable, bindir string, args []string, out output.Outputer, pj *project.Project) (forwardFunc, error) {
```
Comment (Naatan) [inline cmd/state/forward.go:32]: IIRC the forwarder isn't used by the auto updater, it's only used for locked versions. In which case we don't want it to care about tags.

File: cmd/state/main.go [hunk 2]
```diff
@@ -182,7 +182,7 @@ func run(args []string, isInteractive bool, out output.Outputer) error {
 
 	if childCmd != nil && !childCmd.SkipChecks() {
 		// Auto update to latest state tool version, only runs once per day
-		if updated, err := autoUpdate(args, out, pjPath); err != nil || updated {
+		if updated, err := autoUpdate(args, cfg, out, pjPath); err != nil || updated {
```
Comment (Naatan) [inline cmd/state/main.go:185]: If this is cheaper to do for the release branch then I'm on board but I'd prefer we don't pass the cfg around so much and instead pass around the thing we actually want, ie. the tag.

File: installers/install.ps1 [hunk 3]
```diff
@@ -185,10 +185,14 @@ function warningIfadmin() {
```
Comment (Naatan) [inline installers/install.ps1:185]: We don't need to modify these installers on the release branch, as by the time this lands it'd be the ones on the current beta branch that'd end up being used.

That said, we need to revisit our legacy-installers to fully address this. I'll file a story for that.

Comment (Naatan) [reply installers/install.ps1:185]: https://www.pivotaltracker.com/story/show/179305221

**Review by Naatan — CHANGES_REQUESTED** [review]

File: installers/install.ps1 (hunk 3 above)
Comment (mdrohmann) [reply installers/install.ps1:185]: Yes, you are right.  The installers are by now almost the same though, only that the `legacy-install` scripts always require a version.  So, it shouldn't be too complicated...

**Review by Naatan — APPROVED** [review]

**Pull request merged**